src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -852,6 +852,84 @@ protected theorem extractLsb_ofNat (x n : Nat) (hi lo : Nat) :
     (extractLsb' start len x).getLsbD i = (i < len && x.getLsbD (start+i)) := by
   simp [getLsbD, Nat.lt_succ]
 
+/--
+Get the most significant bit after `extractLsb'`. With `extractLsb'`, we extract
+a `BitVec` `x'` with length `len` from `BitVec` `x`, starting from the element
+at position `start`. The function `getMsb` extracts a bit counting from the most
+significant bit. Assuming certain conditions, `x.(extractLsbD' {w} start len).getMsbD i`
```

Comment:
What is the `{w}` here? Is this trying to say that `x : BitVec w`? If so, just say that. Alternatively, if you are trying to specify an implicit argument, just use `@` notation.

Reply:
I switched to @ notation.